tf-deployer/vsphere_named_machines.yaml

```diff
@@ -61,11 +61,16 @@ items:
           $${startup_script}
         path: /tmp/master.sh
         permissions: '0755'
+      - content: |
```

Comment:
Don't tie to CNI to the machine spec here. The templates here define the VM, not the software installed on it.

Reply:
agree this is not a good choice, but looks like the deployer don't have any other solution for file-copy yet. I would like to limit the scope of this PR. If we really don't want to put the copying in here, we have to install online by now.